submarine-workbench/workbench-web/src/app/pages/workbench/environment/environment-home/environment-list/environment-list.component.html

```diff
@@ -0,0 +1,62 @@
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one
+  ~ or more contributor license agreements.  See the NOTICE file
+  ~ distributed with this work for additional information
+  ~ regarding copyright ownership.  The ASF licenses this file
+  ~ to you under the Apache License, Version 2.0 (the
+  ~ "License"); you may not use this file except in compliance
+  ~ with the License.  You may obtain a copy of the License at
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+<nz-table id="environmentTable" nzBordered #basicTable [nzData]="environmentList" [nzNoResult]="'No data'">
+  <thead>
+    <tr>
+      <th>Environment Name</th>
+      <th>Docker image</th>
+      <th>Kernel name</th>
+      <th>Kernel channel</th>
+      <th>Kernel dependencies</th>
+      <th>Action</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr *ngFor="let data of basicTable.data; let i = index">
+      <td>{{ data.environmentSpec.name }}</td>
+      <td>{{ data.environmentSpec.dockerImage }}</td>
+      <td>{{ data.environmentSpec.kernelSpec.name }}</td>
+      <td>
+        <dl *ngFor="let channel of data.environmentSpec.kernelSpec.channels">
+          <dt>{{ channel }}</dt>
+        </dl>
+      </td>
+      <td>
+        <dl *ngFor="let dependency of data.environmentSpec.kernelSpec.dependencies">
+          <dt>{{ dependency }}</dt>
+        </dl>
+      </td>
+      <td>
+        <a
+          *ngIf="data.environmentSpec.name != 'notebook-env'"
```

Comment:
1. Why does the default environment cannot be deleted?

Reply:
@ByronHsu  Hi, because it is used to create notebook.

Reply:
I think it is not appropriate to constrain which environment cannot be deleted in the frontend code. The logic should be written in the backend. You can remove it first. How do you think, @pingsutw ?